docs/admin/provisioners.md

```diff
@@ -331,13 +315,17 @@ will use in concert with the Helm chart for deploying the Coder server.
          value: "https://coder.example.com"
      replicaCount: 10
    provisionerDaemon:
+     # NOTE: in older versions of the Helm chart, it is required to set this to an empty string.
+     pskSecretName: ""
```

Comment:
review: PR to address this workaround is here: https://github.com/coder/coder/pull/15417